Source/Fuse.Scripting.JavaScript/ThreadWorker.uno

```diff
@@ -74,26 +74,21 @@ namespace Fuse.Scripting.JavaScript
 				Fuse.Diagnostics.UnknownException( "ThreadWorked failed", e, this );
 				_exceptionQueue.Enqueue(e);
 			}
-
-			if (_context != null)
-				_context.Dispose();
```

Comment:
This seems wrong. Shouldn't we unconditionally Dispose here instead? Otherwise we'll *never* dispose, which seems like a bad choice...